core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -198,17 +198,20 @@ trait EntityDecoderInstances {
       override def consumes: Set[MediaRange] = Set.empty
     }
 
-  implicit def binary[F[_]: Effect]: EntityDecoder[F, Chunk[Byte]] =
+  implicit def binary[F[_]: Effect]: EntityDecoder[F, Segment[Byte, Unit]] =
     EntityDecoder.decodeBy(MediaRange.`*/*`)(collectBinary[F])
 
+  implicit def binaryChunk[F[_]: Effect]: EntityDecoder[F, Chunk[Byte]] =
+    EntityDecoder.decodeBy(MediaRange.`*/*`)(collectBinary[F]).map(_.force.toChunk)
```

Comment:
this should just map on the previous entity decoder.

Reply:
Does this even need to exist?